src/networks/calibnet/mod.rs

```diff
@@ -20,6 +20,12 @@ use super::{
     parse_bootstrap_peers,
 };
 
+/// Well-known network names.
+pub const NETWORK_COMMON_NAME: &str = "calibnet";
+/// Network name as defined in the genesis block. Refer to [`crate::networks::network_names`] for
```

Comment:
I might be missing something but where exactly is `crate::networks::network_names` for additional info?